src/database.ts

```diff
@@ -1388,6 +1536,90 @@ class Database extends GrpcServiceObject {
     });
     return waitForSessionStream;
   }
+
+  restore(backupPath: string): Promise<RestoreDatabaseResponse>;
+  restore(
+    backupPath: string,
+    options?: CallOptions
+  ): Promise<RestoreDatabaseResponse>;
+  restore(backupPath: string, callback: RestoreDatabaseCallback): void;
+  restore(
+    backupPath: string,
+    options: CallOptions,
+    callback: RestoreDatabaseCallback
+  ): void;
+  /**
+   * @typedef {array} RestoreDatabaseResponse
+   * @property {Database} 0 The new {@link Database}.
+   * @property {Operation} 1 An {@link Operation} object that can be used to check
+   *     the status of the request.
+   * @property {object} 2 The full API response.
+   */
+  /**
+   * @callback RestoreDatabaseCallback
+   * @param {?Error} err Request error, if any.
+   * @param {Database} database The new {@link Database}.
+   * @param {Operation} operation An {@link Operation} object that can be used to
+   *     check the status of the request.
+   * @param {object} apiResponse The full API response.
+   */
+  /**
+   * Restore a backup into this database.
+   *
+   * When this call completes, the restore will have commenced but will not
+   * necessarily have completed.
+   *
+   * @param backupPath the path of the backup to restore.
+   * @param {object} [options] Configuration object.
```

Comment:
Should be documented as `gaxOptions` with a link to the CallOptions page from the gax documentation.

Reply:
https://github.com/googleapis/nodejs-spanner/blob/f8ad0ece28e99dd9c6d7e985de56bd1b376f4712/src/batch-transaction.ts#L106

Reply:
Sorry, I missed this when I looked over the files again.